src/components/Navbar.js

```diff
@@ -1,26 +1,31 @@
 import React from 'react';
 import { Outlet, Link } from 'react-router-dom';
+import { FaUserAlt } from 'react-icons/fa';
+
+// stylesheet
+import './Navbar.css';
 
 function Navbar() {
   return (
     <div className="navbar">
       <div className="title-div">
         <h2 className="title">BOOKSTORE CMS</h2>
-      </div>
-      <div className="menu">
-        <nav>
-          <ul className="menu-list">
-            <li className="list-item">
-              <Link to="/">Books</Link>
-            </li>
-            <li className="list-item">
-              <Link to="/categories">Categories</Link>
-            </li>
-          </ul>
-        </nav>
+        <div className="menu">
+          <nav>
+            <ul className="menu-list">
+              <li className="list-item">
+                <Link to="/">Books</Link>
+              </li>
+              <li className="list-item">
+                <Link to="/categories">Categories</Link>
+              </li>
+            </ul>
+          </nav>
 
-        <Outlet />
+          <Outlet />
+        </div>
       </div>
+      <FaUserAlt className="icon" />
     </div>
```

Comment:
- [ ] Kindly improve the font of the navbar section to match the design.

  Link to the design: https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/sneak_peek_v2_0.md

  ![siisos](https://user-images.githubusercontent.com/54848363/230662695-2d07eee5-7fe2-4eb4-9cb6-89275e5ddee4.jpg)

Reply:
Ok well noted